internal/winapi/memory.go

```diff
@@ -0,0 +1,21 @@
+package winapi
+
+// HANDLE GetProcessHeap();
+//
+//sys GetProcessHeap() (procHeap windows.Handle, err error) = kernel32.GetProcessHeap
+
+// DECLSPEC_ALLOCATOR LPVOID HeapAlloc(
+// 	HANDLE hHeap,
+// 	DWORD  dwFlags,
+// 	SIZE_T dwBytes
+// );
+//
+//sys HeapAlloc(hHeap windows.Handle, dwFlags uint32, dwBytes uintptr) (lpMem uintptr, err error) = kernel32.HeapAlloc
+
+// BOOL HeapFree(
+// 	HANDLE                 hHeap,
+// 	DWORD                  dwFlags,
+// 	_Frees_ptr_opt_ LPVOID lpMem
+// );
```

Comment:
are these definitions meant to stay in?

Reply:
Yes I feel we should do these for all of our bindings. Makes it easier when going to make future additions to winapi to see what go types map to what windows types. Avoids unnecessary web searches

Reply:
fair enough, I have no problem with that